api/models.py

```diff
@@ -24,15 +24,26 @@ def __str__(self) -> str:
         return self.english
 
 
-class MemoryGameSession(models.Model):
+# Game Sessions
+class BaseGameSession(models.Model):
```

Comment:
Yeeees, very nice.

Can we add `duration` to BaseGameSession (and update the RaceGameSession creation code) so we also have duration for races?

Reply:
Good idea, will do.